doc-site/vite.config.ts

```diff
@@ -0,0 +1,11 @@
+import { defineConfig } from 'vite';
+import path from 'path';
+
+export default defineConfig({
+  resolve: {
+    alias: {
+      '@assets': path.resolve(__dirname, './assets'),
```

Comment:
bruker vi det?

Reply:
Ikke længre, testade att bruka den for att undvika længre paths men vi kan fjerne den.